src/main/java/com/google/alpollo/SongCounter.java

```diff
@@ -0,0 +1,22 @@
+package com.google.alpollo;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Index;
+
+/** Represents the structure of the song to save search counter in the database. */
+@Entity
+public class SongCounter extends Song {
```

Comment:
Why not add searchCounter as a field to the Song class instead of creating this new class?
This inheritance also does not make sense from an Object Oriented point of view. SongCounter is not a particular sub category of Song.